src/maintainer/knowledge_base.rst

```diff
@@ -554,45 +554,70 @@ Special packages
 ================
 
 BLAS
-----
+....
 
-.. todo::
+If a package needs one of BLAS, CBLAS, LAPACK, LAPACKE, use the following in the
+host of the recipe,
+
+.. code-block:: yaml
+    requirements:
+      host:
+        - libblas
+        - libcblas
+        - liblapack
+        - liblapacke
+
+Note that you should specify only the libraries the package needs. (i.e. if the package
+doesn't need LAPACK, remove liblapack and liblapacke)
+
+At recipe build time, above requirements would download the NETLIB's reference
+implementations and build your recipe against those. At runtime by default the following packages
+will be used,
+
+.. code-block:: yaml
+    - openblas   # [not win]
+    - mkl        # [win]
 
-  The information regarding the blas package may now be outdated and needs checking.
```

Comment:
What do you think about moving the following paragraphs into a new section: User Documentation -> Configuration -> Choose BLAS Implementation?

Reply:
I don't want to break up the building package and using to separate section. (Separate subsections within the same section is fine.) Let me know if you have any suggestions.

Reply:
Fair enough. If we decide later to start a config section, we can always cross reference here. For a better structure I'd put it in a box or use subsection.